app/presenters/auction_presenter.rb

```diff
@@ -140,6 +145,14 @@ def url
     "#{root_url}/auctions/#{id}"
   end
 
+  def eligibility
```

Comment:
This should probably be delegated to the Eligibility object itself (maybe with a name like `eligibility_label` or such